regression/rt_gk_bgk_3x2v_p1.c

```diff
@@ -427,16 +428,17 @@ main(int argc, char **argv)
       .temp = evalTempElcInit,
       .ctx_temp = &ctx,
     },
+
+    .correct_all_moms = true, 
+    .use_last_converged = true, 
+    .iter_eps = 1e-12,
+    .max_iter = 10,     
```

Comment:
These inputs should be consolidated into a `maxwellian_correction_inp` struct, rather than placed loosely here, and then we can add that struct as an input to ICs, collisions, sources, etc (wherever the user wants to make corrections).